src/base/static/scss/_forms.scss

```diff
@@ -1,4 +1,5 @@
 // Forms - Selectors
+@import "../../../../node_modules/@mapseed/accessible-autocomplete/dist/accessible-autocomplete.min";
```

Comment:
I understand that this is not ideal, and we can simplify these paths by moving our CSS build step over to webpack, as described in my comment here: https://github.com/mapseed/platform/pull/796/files#r132825501